integration/common/openlineage/common/provider/bigquery.py

```diff
@@ -141,40 +162,70 @@ def get_facets(self, job_id: str) -> BigQueryFacets:
                     )
                 }
             )
-        return BigQueryFacets(run_facets, inputs, output)
-
-    def _get_output_statistics(
-        self, properties
-    ) -> Tuple[BigQueryJobRunFacet, Optional[BigQueryStatisticsDatasetFacet]]:
-        stages = get_from_nullable_chain(properties, ["statistics", "query", "queryPlan"])
-        json_props = json.dumps(properties)
-
-        if not stages:
-            if get_from_nullable_chain(properties, ["statistics", "query", "statementType"]) in [
-                "CREATE_VIEW",
-                "CREATE_TABLE",
-                "ALTER_TABLE",
-            ]:
-                return BigQueryJobRunFacet(cached=False), None
-
-            # we're probably getting cached results
-            if get_from_nullable_chain(properties, ["statistics", "query", "cacheHit"]):
-                return BigQueryJobRunFacet(cached=True), None
-            if get_from_nullable_chain(properties, ["status", "state"]) != "DONE":
-                raise ValueError("Trying to extract data from running bigquery job")
-            raise ValueError(f"BigQuery properties did not have required data: queryPlan - {json_props}")
-
-        out_stage = stages[-1]
-        out_rows = out_stage.get("recordsWritten", None)
-        out_bytes = out_stage.get("shuffleOutputBytes", None)
+        return BigQueryFacets(
+            run_facets=run_facets,
+            inputs=list({i.name: i for i in inputs if i}.values()),
+            outputs=self._deduplicate_outputs(outputs),
+            output=outputs[0] if len(outputs) == 1 else None,
```

Comment:
Couldn't this default to first element if exists when integration uses older `output` attribute because of no update there?

Reply:
I think that we should somehow indicate to people that there are more than one output. If there are multiple outputs and somebody is using `output` - they will still receive something and may not be aware that there are more. My idea was - if they see no output at all, they will look into it, probably discover `outputs` and update the code. I'm open to changes, no hard arguments here.

Reply:
I don't really like the idea of limiting users' experience just to force them to use newer libraries. It wouldn't be the case if the change was only for `SCRIPT` type jobs which don't work so far. This on the other hand breaks output on purpose for any other job.

Reply:
I'm not sure i understand. At the moment, you can only receive a single output dataset from a single BQ job (one job == one statement). After my change in this PR, for all jobs other than `SCRIPT` you will still receive a single output dataset (as a dataset under `output` and as a list with one dataset under `outputs`). For `SCRIPT` however, if there are multiple outputs, you have to use `outputs` to get all of them and the output will be empty (just like right now, the script job returns no input/outputs). So as far as i understand, the behaviour won't change (apart from `output` being deprecated).

Reply:
Sorry, you're right!
I don't have strong opinion though now. Maybe let's not leave first output for `SCRIPT` jobs as this may lead to some strange behaviours and questioned results. Could you please leave some comment at this place to explain the logic?

Reply:
Sure. Multiple outputs are there for more complex scripts that create or insert data into multiple tables (multiple INSERT INTO, CREATE statements etc.) - then we will receive multiple output datasets and we need to use `outputs` because the `output` will be empty. However, the script can also be a single statement (f.e. CREATE TABLE AS) with some variables declared at the top (f.e. DECLARE var1 ...) - in this case, we will receive a single output dataset. Even though the statement type is `SCRIPT` we can treat it as a regular statement and also make this single output available under `output`. This way, even if somebody doesn't know that they are using a `SCRIPT` job, because they are simply running a single statement with variables, will still be able to use `output` until removed from deprecation. I think it should not lead to any unexpected behaviours, but maybe i am missing something.

Reply:
Added a [comment](https://github.com/OpenLineage/OpenLineage/pull/2564/files#diff-3be24ccd3c1a25a965537f442716bdf033abcd9346056db006b1922466f81487R167-R170) in the file as well, let me know if it requires some changes.

Reply:
Looks great, thanks!